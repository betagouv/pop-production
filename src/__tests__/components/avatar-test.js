import React from "react";
import Avatar from "../../components/avatar";
import renderer from "react-test-renderer";

test("Avatar displays two letters (FB)", () => {
  const component = renderer.create(<Avatar email="foo.bar@example.org" />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
