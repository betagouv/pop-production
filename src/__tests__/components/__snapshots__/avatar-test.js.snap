// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar displays two letters (FB) 1`] = `
<span
  className="avatar"
>
  <div
    className="Initials"
  >
    FB
  </div>
</span>
`;
